core-bundle/src/Resources/contao/dca/tl_image_size.php

```diff
@@ -103,7 +103,7 @@
 	// Palettes
 	'palettes' => array
 	(
-		'default'                     => '{title_legend},name,width,height,resizeMode,zoom;{expert_legend},cssClass,densities,sizes'
+		'default'                     => '{title_legend},name,width,height,resizeMode,zoom;{expert_legend},cssClass,densities,sizes,skipIfDimensionsMatch'
```

Comment:
Note to myself: check if the field needs to be in a separate section after the PR has been merged.